src/utils/IOUtil.cpp

```diff
@@ -130,6 +130,11 @@ utils::getUnitDimension ( std::string const & record_name )
         {openPMD::UnitDimension::I, -1.},
         {openPMD::UnitDimension::T, -2.}
     };
+    else if( record_name == "spin" ) return {
+        {openPMD::UnitDimension::L,  2.},
+        {openPMD::UnitDimension::M,  1.},
+        {openPMD::UnitDimension::T, -1.}
+    };
```

Comment:
According to @pots007 the community would normally used some normalisation here, probably also in the input. If so, the units should be fixed. Let's still merge this PR so we can proceed, and come back to it upon finalisation.